setup.py

```diff
@@ -102,7 +102,7 @@ def run(self):
       data_files=data_files,
       cmdclass=cmdclass,
       # rosinstall dependency to be kept in order not to break ros hydro install instructions
-      install_requires=['vcstools>=0.1.37', 'pyyaml'],
+      install_requires=['vcstools>=0.1.37', 'pyyaml', 'pyaml'],
```

Comment:
I had not noticed before. Is there any meaningful difference between pyyaml and pyaml that is important to the feature? I looked at : https://pypi.python.org/pypi/pyaml/15.8.2
If possible, less dependencies are better.